libs/sdk-ui-dashboard/src/_staging/layout/coordinates.ts

```diff
@@ -41,9 +41,19 @@ export const areLayoutPathsEqual = (
     if (path1.length !== path2.length) {
         return false;
     }
-    return path1.every(
-        (path, index) =>
-            path.sectionIndex === path2[index].sectionIndex && path.itemIndex === path2[index].itemIndex,
+    return path1.every((path, index) => areSameCoordinates(path, path2[index]));
+};
+
+export const areSameCoordinates = (
```

Comment:
There already is method for this at `libs/sdk-ui-dashboard/src/_staging/layout/coordinates.ts`: `areLayoutPathsEqual`